crates/holochain/src/core/workflow/sys_validation_workflow/validation_query.rs

```diff
@@ -46,6 +50,30 @@ async fn get_ops_to_validate(
                 DhtOp.validation_stage IS NULL
                 OR DhtOp.validation_stage = 0
             )
+
+            UNION ALL
+
+            SELECT
+            -- Note the odd field name, this is mapped later based on the DhtOp.type
+            Warrant.blob as action_blob,
+            Warrant.author as author,
+            NULL as entry_blob,
+            DhtOp.type as dht_type,
+            DhtOp.hash as dht_hash,
+            DhtOp.num_validation_attempts,
+            DhtOp.op_order
+            FROM DhtOp
+            JOIN Warrant ON DhtOp.action_hash = Warrant.hash
+
+            WHERE
+            DhtOp.when_integrated IS NULL
+            AND DhtOp.validation_status IS NULL
+            AND (
+                DhtOp.validation_stage IS NULL
+                OR DhtOp.validation_stage = 0
+            )
```

Comment:
Sys validation was claiming to validate warrants but that can't have been tested because the database query was broken by the refactor to split warrants out of the action table... There were tests for issuing warrants but not validating them.